guides/source/layout.html.erb

```diff
@@ -14,6 +14,13 @@
   <script src="javascripts/turbolinks.js" data-turbolinks-track="reload"></script>
   <script src="javascripts/guides.js" data-turbolinks-track="reload"></script>
   <script src="javascripts/responsive-tables.js" data-turbolinks-track="reload"></script>
+  <meta property="og:title" content="<%= yield(:page_title) || 'Ruby on Rails Guides' %>" />
```

Comment:
Does this mean that any page has no title?

Reply:
Not sure. I wrote this code same as existing code:

https://github.com/rails/rails/blob/29bb990f97816f3e0fb086ec9985983c51a56a96/guides/source/layout.html.erb#L6

But, as far as I saw, all page should have title. 🤔

Reply:
If so, I'd prefer to remove the redundant ` || 'Ruby on Rails Guides'`.

Reply:
🆒  I'm gonna remove it 🔪

Reply:
@kamipo Did it!